packages/extension-base/src/background/handlers/State.ts

```diff
@@ -362,26 +285,24 @@ export default class State {
     if (authUrls[idStr]) {
       assert(authUrls[idStr].authorizedAccounts || authUrls[idStr].isAllowed, `The source ${url} is not allowed to interact with this extension`);
 
-      return {
+      return respond({
         authorizedAccounts: [],
         result: false
-      };
+      });
```

Comment:
I don't think we care about the results here :thinking:

Reply:
Turned out we care - the response is passed back to the initiating tab and further consumed outside of the Signer flow.